bootloader/wscript

```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```

Comment:
> Why do we need to set WINRC_SRC_F to an empty string?

Because with gcc, the include files are prefixed with `-i` whereas with visual studio they are not.